site/class-1-load/vriesm060/vriesm060/dinners.txt

```diff
@@ -0,0 +1,9 @@
+Day:
```

Comment:
These files are in `site/class-1-load/vriesm060/vriesm060`, but they should be in `site/class-1-load/vriesm060`!